content/metadata.yaml

```diff
@@ -26,6 +26,17 @@ author_info:
     email: slochower@gmail.com
     affiliations:
       - Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego
+  -
+    github: vsmalladi
+    name: Venkat S. Malladi
+    initials: VSM
+    orcid: 0000-0002-0144-0564
+    twitter: katatonikat
```

Comment:
@vsmalladi is this a typo?  Should it be `katatonikkat`?

Reply:
Yes it should be katatonikkat.
 Also affiliations is spelt wrong